docs/http/traffic-policy/actions/jwt-validation.mdx

```diff
@@ -4,14 +4,17 @@ import { ExampleHTTP } from "/examples/actions/jwt-validation.mdx";
 
 ## Overview
 
-The JWT Validation action allows or denies traffic based on validation of the JSON Web
-Token (JWT) provided within the request that was initiated to your ngrok endpoints. You
-will define the claims used to validate the token in conjunction with verifying it has
-been properly signed by the issuer.
+The JWT Validation action allows you to configure ngrok to validate JSON Web Tokens (JWT) in our cloud before routing the traffic to your origin service. You will need to tell ngrok where to find the token, as well as the claims and algorithms used to verify it's authenticity.
```

Comment:
This implies to me that it's only available as a cloud-initiated action, but I don't think we prevent this from being used on the agent-initiated side. What is actually expected?

Reply:
I think the intent is to say that JWT validation will occur in ngrok's cloud service rather than on your server/backend/infrastructure. I don't think the intent is to limit it to cloud-initiated actions vs agent-intiated. Is there a way to phrase that which wouldn't cause confusion with policy action verbiage?

Reply:
Adding 
>  When configuring this action in a locally running agent, it is pushed to our global network which secures traffic before it hits your network. 

to help clarify?